twitter/View Controllers/TimelineViewController.m

```diff
@@ -73,5 +74,16 @@ - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(N
 {
     return self.tweetsArray.count;
 }
-
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender{
+    UINavigationController *navigationController = [segue destinationViewController];
+    ComposeViewController *composeController = (ComposeViewController*)navigationController.topViewController;
+    composeController.delegate = self;
```

Comment:
Need to check that this is a `ComposeViewController`. It can crash if it's a different view controller that doesn't have a `setDelegate` method.

Reply:
yeah it has a delegate for the ComposeViewController

Reply:
True. But should have a type check. Like “composeController isMemberOfClass: ComposeViewController.class”